eventually/static/src/containers/item/answer.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
name of files with components should start with the uppercase letter.

Reply:
Components start with Uppercase, files camelCase

Reply:
I found that advice in following articles: (https://daveceddia.com/react-project-structure/, https://survivejs.com/react/advanced-techniques/structuring-react-projects/) and also WebStror IDE notifies me about it =)